exec-analysis/src/main/java/com/capitalone/dashboard/exec/collector/DashBoardCollectorItemMapBuilder.java

```diff
@@ -77,6 +87,41 @@ public void setItems(List<String> items) {
                 return dashboardCollectorItem;
             };
 
+    public static final List<DashboardCollectorItem> getPipelineDashboardCollectorItems(Dataset<Row> ds) {
```

Comment:
Can you explain why this is needed? Pipelines have collector items just like the other widgets. As such only a new collector class should needed. If you need to get the collector items any other way there is likely a problem with the association of the collector item to the dashboard. In that case it should be fixed there instead of here.

Reply:
I see what is going on here. I am resolving this for now